jobs/ci_open_jenkins/build/financial-transactions.groovy

```diff
@@ -1,4 +1,6 @@
-package ci_open_jenkins.build
+/*package ci_open_jenkins.build
```

Comment:
Can't we just delete the file? If there's a problem we can always revert the commit. It saves making another PR after this to remove it.